recipes/librdmacm-cos6-x86_64/meta.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: librdmacm-cos6-x86_64
+  version: 1.0.21
+
+source:
+  - url: http://mirror.centos.org/centos/6.10/os/x86_64/Packages/librdmacm-1.0.21-0.el6.x86_64.rpm
+    sha256: 39887f875e4e34964d842a519805de08c39f198862f3dc4f81bb9b804660fa34
+    folder: binary
+  - url: http://vault.centos.org/6.10/os/Source/SPackages/librdmacm-1.0.21-0.el6.src.rpm
+    folder: source
```

Comment:
Currently there is no good way to determine the checksum of the source RPM. Please see issue ( https://github.com/conda/conda-build/issues/3552 ) for details.

Reply:
I've gone ahead and dropped the source RPM as it is not used.